actions/setup-serverless/action.yml

```diff
@@ -0,0 +1,14 @@
+# This file is the sharable action with build,unittest, integration test, code coverage reports.
```

Comment:
please remove this function out. we don't want to have infra provisioning is performed from GitHub Action

Reply:
We need to use serverless setup to deploy lambda. I discussed this with @johnseq-iterative and clearly mentioned that infrastructure provisioning code should not be part of this pipeline. If you see the serverless templates [here](https://github.com/Iterative-Scopes/cdp-dynamodb-spike/blob/main/cdp-video-metadata/procedure-qb-export/serverless-dev.yml) only the lambda specific roles and events are getting created from the template. We are expecting the other infra should already be in place before we run this pipeline.

Reply:
@is-ttang Avik and I did discuss this.  My understanding is the plan is to keep infra+code provisioning together until we are ready to separate them out.  That keeps the CICD responsibility squarely on one team while we wait to e.g. move to gitlab.  Does that work as an interim solution?

Reply:
Sorry -- reading this more closely.  @is-achakraborty this makes sense.  We'll split out as much as we can from our templates and create out of band. That's fine.

Reply:
Yes. but, you can run the SLS provisioning separately. it requires have more privilege for GitHub to access AWS  to provision. Second, you just need to run ONCE, NOT run lambda update every time.  At moment, GitHub OpenID connect scope is locked down to ECR publish and Lambda S3 write

Reply:
`Separately` means out of the pipeline? If yes then I think no meaning in using this pipeline anymore. 
Yes we need the required AWS permissions for the pipeline to work. We can sit together and decide on the minimal subset of permission required.